dbtmetabase/metabase.py

```diff
@@ -404,14 +434,192 @@ def build_metadata_lookups(
 
         return table_lookup, field_lookup
 
+    def extract_exposures(
```

Comment:
Would be nice to have header comments for these big important functions. How they work, inputs, outputs, etc.

Reply:
done